### ISSUE TITLE
NPC Type Arrow throws IllegalArgumentException

### ISSUE THREAD

**Issue by xxcodianxx** [CitizensDev/Citizens2#2799]
### `/version` output

```txt
This server is running CraftBukkit version git-PaperSpigot-445 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)
```

### `/version Citizens` output

```txt
Citizens version 2.0.29-SNAPSHOT (build 2546)
```

### Log file

https://paste.denizenscript.com/View/96459

### config.yml

https://paste.denizenscript.com/View/96460

### saves.yml

(completely empty)

### Description

Citizens throws an exception whenever you try to set the NPC type to an arrow. The entity type is validated by the command interface, the error is most likely internal.

**How To Reproduce**
1. Load any fresh 1.8 server (here I'm using the unforked paper 1.8.8 build).
2. /npc create test
3. /npc type arrow
4. :boom:

**Comment by mcmonkey4eva** [comment]
https://wiki.citizensnpcs.co/Minecraft_1.8

**Comment by xxcodianxx** [comment]
So no chance of this ever getting fixed for 1.8?

**Comment by xxcodianxx** [comment]
@fullwall Thank you for the attempted fix, but even after c03ffef94025408f773ce824aaa96bafe00260e1 the issue persists in exactly the same way.

The ArrowController class you added is being correctly included in the jarfile from the CI, but maybe the controller isn't being used for the Arrow type for some reason?

Let me know if you can reproduce this, and find out where the bug is! :+1:

**Comment by xxcodianxx** [comment]
Hello again, this was fixed for arrows about 14 days ago (5b274ad7a7bfa58802961275ff39dce983cb312f), but I have noticed that the entity type `SPLASH_POTION` is also not implemented for 1.8, even though the very similar `THROWN_EXP_BOTTLE` is. 

Is it simple and/or possible to implement the fix for splash potions in the 1.8 NMS? 
Thank you for the quick actions so far. :+1:
